### ISSUE TITLE
Phoenix And Phoenix Feathers

### ISSUE THREAD

**Issue by EternaDash** [Sollace/Unicopia#405]
Well #397 Was A Disaster, So Here's Episode One Of Me Ripping Apart It's Corpse. Today, We're Focusing On The Phoenix Mob.

1: Mob Itself
=

Name: Phoenix
Health: 10
Damage: Doesn't Attack
Nature: Passive
Drops: 0-1 Phoenix Feather
Special: Can Fly, Can Be Tamed And Bred With Fish, Rarely Sheds Phoenix Feathers If Tamed
Spawns In: Overworld (rare)

2: What Is A Phoenix Feather Used For?
=

A Few Things, Actually. Thanks For (hopefully) Asking.

Phoenix Arrows: Fire Arrows All Without The Need For The Flame Enchant. Incompatible With Infinity Enchantment. Can Be Found In Chests Rarely As Loot. Crafted Like A Normal Arrow But With A Phoenix Feather. If Used With A Flame Bow, It Will Deal Extra Damage And Ignite For Longer. Skeletons Can Rarely Fire These Instead Of Regular Arrows On Hard.

Potion Of Inferno: Made With Brewing With A Phoenix Feather On A Potion Of Fire Resistance. Lasts About 30 Seconds, And Creates A (flat) Ring Around The Player That Ignites Enemies On Fire While Knocking Them Back With Some Upfront Damage. You Can Use A Signed Bangle of Comradery To Make Your Allies Immune To The Burning Effects, Just Keep Away From Your Livestock Or Pets While Active. This Would *Not* Make You Immune To Fire Damage, And Fire-Immune Mobs Or Ponies With Fire Resistance Will Take Zero Damage. Inspired By The [Terraria Potion](https://terraria.wiki.gg/wiki/Inferno_Potion) Of Near-Same Name.

Phoenix Banner Pattern: Banner Pattern Made With A Phoenix Feather That, Unsurprisingly, Looks Like A Phoenix.

Potion Of Rebirth: Made With Brewing With A Phoenix Feather On A Potion Of Regeneration. Lasts 5-7 Seconds, But Dying With The Effect Will Remove It And Act As A Totem Of Undying. Cannot Be Made Into A Lingering Potion, Because That Would Just Be An Immortality Machine.

Recipe For Totem Of Undying: 5 Gold Ingots, 1 Phoenix Feather, 3 Diamond. Totems Are Already Dirt Cheap With Raid Farms, So This Doesn't Seem Like A Horrible Idea. Maybe Add In A Netherite Ingot If You Want.

3: Extras
=

Spawns As Treasure: Can Be Found In Buried Treasure Chests, Dungeon Chests, Desert Temple Chests, And Stronghold Chests Rarely. And Spawns Uncommonly In End City Chests. Could Also Just Forgo The Mob Itself And Only Have The Phoenix Feather, Instead Adding The Mob Later Down The Line So Ponies Can Play Around With Some New Toys Earlier.

**Comment by OakPlatinum** [comment]
Actually, you can attach it to #229 and mention Sollace.
A phoenix pet is really awesome, I think. Making it fight enemies is more reasonable. And is health of 10 (which is 5 hearts in game UI) too low?

**Comment by OakPlatinum** [comment]
Plus, a phoenix is just better than any of vanilla pets. Vanilla pets are sometimes silly, falling into the lava, the void or just getting stuck in walls. However, a phoenix pet is much greater because it's immortal (can respawn after a specific cooldown) and fire-immune. So I suggest it be VERY RARE, not RARE.
And only in a new structure called Castle of the Two Sisters or just in Everfree Forest biome suggested in #138 can a phoenix or its egg be found?
In conclusion, it should be a great challenge for a player to find a phoenix (or its egg). I bet it would replace Axolotl and be the most adorable pet in the whole game.

**Comment by EternaDash** [comment]
> Actually, you can attach it to #229 and mention Sollace.

Ah, Didn't See That Since It Didn't Show Up When I Searched "Phoenix" In The Issue Searchbar Thing

**Comment by EternaDash** [comment]
First 1.5.0 Milestone Let's Gooooo
